src/server/fs/lib/fs-manager.js

```diff
@@ -26,7 +26,7 @@ var FsExtra = require('fs-extra');
 var express = require('express');
 var walkdir = require('walkdir');
 var async = require('async');
-var _ = require('underscore');
+var _ = require('lodash');
```

Comment:
Because I used the partialRight API which only supported on lodash module.
There was no side effect since lodash fully covered under score APIs